admin/class-helpers.php

```diff
@@ -211,4 +211,21 @@ function ( $rule ) {
 		}
 		return $results;
 	}
+
+	/**
+	 * Do a capability check for the current user to ensure they have the required capability
+	 * to see various widgets or notices.
+	 *
+	 * @since 1.9.3
+	 */
```

Comment:
add @return parameter:

` * @return bool True if the current user has the required capability, false otherwise.`